crates/volta-core/src/tool/package/resolve.rs

```diff
@@ -183,17 +182,14 @@ fn npm_view_command_for(name: &str, version: &str, session: &mut Session) -> Fal
 // fetch metadata for the input url
 fn resolve_package_metadata(package_name: &str, package_info_url: &str) -> Fallible<PackageIndex> {
     let spinner = progress_spinner(&format!("Fetching package metadata: {}", package_info_url));
-    let response_text = attohttpc::get(package_info_url)
-        .send()
-        .and_then(Response::error_for_status)
-        .and_then(Response::text)
+    let response_text = reqwest::blocking::get(package_info_url)
+        .and_then(|resp| resp.error_for_status())
+        .and_then(|resp| resp.text())
```

Comment:
Similarly here, I believe we should be able to keep the same point-free style with `.and_then(Response::error_for_status).and_then(Response::text)` to avoid creating unnecessary closures.